docs/how-to/how-to-use-k8s-with-cri-containerd-and-kata.md

```diff
@@ -125,43 +123,35 @@ $ sudo systemctl daemon-reload
   $ sudo -E kubectl get pods
   ```
 
-## Install a Pod Network
+## Configure Pod Network
 
 A pod network plugin is needed to allow pods to communicate with each other.
+The simplest type of CNI is `bridge`.
+But you can find more about CNI plugins from the [Creating a cluster with `kubeadm`](https://kubernetes.io/docs/setup/independent/create-cluster-kubeadm/#instructions) guide,
+starting from the **Installing a pod network** section.
```

Comment:
It's hard to understand this sentence.

Reply:
@liubin, what about the following text:

```
The simplest type of CNI is `bridge`.
But you can find more about CNI plugins from the [Creating a cluster with `kubeadm`](https://kubernetes.io/docs/setup/independent/create-cluster-kubeadm/#instructions) guide, specifically in the [pod network section](
https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/#pod-network).
```

Reply:
@Tim-Zhang, does that clarify the sentence? If so, @liubin, mind to change it?

I really would like to have this merged. :-)

Reply:
You can find more about the plugin from the [Creating a cluster with `kubeadm`](https://kubernetes.io/docs/setup/independent/create-cluster-kubeadm/#instructions) guide.